File: 220722-CaesarCipher/app/src/test/java/CaesarCipherTest.java
import org.junit.jupiter.api.*;

import static org.junit.jupiter.api.Assertions.*;

class CaesarCipherTest {
  private CaesarCipher caesarCipher = new CaesarCipher();
  @BeforeEach
  void declaration() {
    //CaesarCipher caesarCipher = new CaesarCipher();

  }
  @Test
  void getAlphabet() {
    assertEquals("BC",caesarCipher.getAlphabet("AB", 1, ""));
    assertEquals("a",caesarCipher.getAlphabet("z", 1, ""));
    assertEquals("eFd",caesarCipher.getAlphabet("aBz", 4, ""));
    assertEquals("e F d",caesarCipher.getAlphabet("a B z", 4, ""));
    assertEquals("e F  d",caesarCipher.getAlphabet("a B  z", 4, ""));
    
}

}

File: 220819-caesarCipher/caesarCipher.test.js
import { caesarCipher } from './caesarCipher';

test('caecarCipher', () => {
  expect(caesarCipher('AB', 1)).toBe('BC');
  expect(caesarCipher('CD', 1)).toBe('DE');
  expect(caesarCipher('Z', 1)).toBe('A');
  expect(caesarCipher('z', 1)).toBe('a');
});
